src/main/resources/l10n/JabRef_de.properties

```diff
@@ -2176,187 +2178,188 @@ Search_results_in_database_%0_for_%1=Suchergebnisse_in_Datenbank_%0_für_%1
-Display_keywords_appearing_in_ALL_entries=
-Display_keywords_appearing_in_ANY_entry=
-Fetching_entries_from_Inspire=
-None_of_the_selected_entries_have_BibTeX_keys.=
-Unabbreviate_journal_names=
-Unabbreviating...=
-Usage=
-
-Are_you_sure_you_want_to_reset_all_settings_to_default_values?=
-Reset_preferences=
-
-Ill-formed_entrytype_comment_in_bib_file=
-Clipboard=
-Could_not_paste_entry_as_text\:=
-Do_you_still_want_to_continue?=
-Please_enter_the_desired_name\:=
-SQL_Export=
-This_action_will_modify_the_following_field(s)_in_at_least_one_entry_each\:=
-This_could_cause_undesired_changes_to_your_entries.=
-You_have_entered_an_invalid_or_already_existent_DB_name.=
-
-Disable_highlight_groups_matching_entries=
-Run_field_formatter\:=
-
-Adds_{}_brackets_around_acronyms,_month_names_and_countries_in_%s_to_preserve_their_case.=
-Converts_units_in_%s_to_LaTeX_code.=
-Does_nothing.=
-
-the_given_field=
-
-Table_font_size_is_%0=
-
-
-Internal_style=
-Add_style_file=
-Are_you_sure_you_want_to_remove_the_style?=
-Current_style_is_'%0'=
-Remove_style=
-Select_one_of_the_available_styles_or_add_a_style_file_from_disk.=
-You_must_select_a_valid_style_file.=
-
-Reload=
-
-Capitalize=
-Capitalize_all_words,_but_converts_articles,_prepositions,_and_conjunctions_to_lower_case.=
-Capitalize_the_first_word,_changes_other_words_to_lower_case.=
-Changes_all_letters_to_lower_case.=
-Changes_all_letters_to_upper_case.=
-Changes_the_first_letter_of_all_words_to_capital_case_and_the_remaining_letters_to_lower_case.=
-Cleans_up_LaTeX_code.=
-Completely_the_field.=
-Converts_HTML_code_to_LaTeX_code.=
-Converts_LaTeX_encoding_to_Unicode_characters.=
-Converts_Unicode_characters_to_LaTeX_encoding.=
-Converts_ordinals_to_LaTeX_superscripts.=
-HTML_to_LaTeX=
-LaTeX_cleanup=
-LaTeX_to_Unicode=
-Lower_case=
-Minify_list_of_person_names=
-Normalize_date=
-Normalize_month=
-Normalize_month_to_BibTex_standard_abbreviation.=
-Normalize_names_of_persons=
-Normalize_page_numbers=
-Normalize_pages_to_BibTeX_standard.=
-Normalizes_lists_of_persons_to_the_BibTeX_standard.=
-Normalizes_the_date_to_ISO_date_format.=
-Ordinals_to_LaTeX_superscript=
-Protect_terms=
-Remove_enclosing_braces=
-Removes_braces_encapsulating_the_complete_field_content.=
-Sentence_case=
-Shortens_lists_of_persons_if_there_are_more_than_2_persons_to_"et_al.".=
-Title_case=
-Unicode_to_LaTeX=
-Units_to_LaTeX=
-Upper_case=
-Identity=
+Copy_special=Kopiere_spezial
+
+Abbreviate_journal_names=Kürze_Zeitschriftentitel_ab
+Abbreviating...=Kürze_ab...
+Adding_fetched_entries=Füge_abgerufene_Einträge_hinzu
+Are_you_sure_you_want_to_remove_the_already_existent_SQL_DBs?=Wollen_Sie_wirklich_die_bereits_bestehenden_SQL-Datenbanken_entfernen?
+Display_keywords_appearing_in_ALL_entries=Zeige_Schlüsselwörter_die_in_ALLEN_Einträgen_vorkommen
+Display_keywords_appearing_in_ANY_entry=Zeige_Schlüsselwörter_die_in_ALLEN_Einträgen_vorkommen
+Fetching_entries_from_Inspire=Rufe_Einträge_von_Inspire_ab
```

Comment:
I thought those fixed things like names were removed from the Strings? @oscargus @matthiasgeiger